lib.rs

```diff
@@ -0,0 +1,86 @@
+
+pub mod filerw {
```

Comment:
which naming convention is this?? Read about diff naming conventions...and use snake case for method and variable naming